cloudformation/content-authorisation-proxy.cf.yaml

```diff
@@ -144,7 +144,7 @@ Resources:
             - logs:*
             Resource:
             - arn:aws:logs:*:*:*
-      - PolicyName: dynamoTable
+      - PolicyName: unEncryptedDynamoTable
```

Comment:
we keep the permission to the old table just to make we have no downtime in the time between cfn update and deployment. It would probably not be an issue but we do this just as an extra precaution